client/styles/FEAuth.module.css

```diff
@@ -0,0 +1,31 @@
+.main{
```

Comment:
No space between the selector and opening brace, I guess autoformatting doesn't apply to CSS yet